config.py.template

```diff
@@ -3,38 +3,43 @@
 from __future__ import unicode_literals
 
 DEBUG = False
-SECRET_KEY = ''  # A secret key used by Flask for signing. KEEP THIS SECRET! (e.g. 'JbwLnLgsfxDInozQudc6IFPe0eYecW8f')
+
+# A secret key used by Flask for signing. KEEP THIS SECRET!
+# (e.g. 'JbwLnLgsfxDInozQudc6IFPe0eYecW8f')
+SECRET_KEY = "CHANGEME"
 
 # Chrome 80 SameSite=None; Secure fix
 SESSION_COOKIE_SECURE = True
 SESSION_COOKIE_SAMESITE = "None"
 
-ALLOWED_CANVAS_DOMAINS = []  # A list of domains that are allowed to use the tool. (e.g. ['example.com', 'example.edu'])
+# A list of domains that are allowed to use the tool. (e.g. ['example.com', 'example.edu'])
+ALLOWED_CANVAS_DOMAINS = []
 
-CANVAS_URL = ''  # Canvas Instance URL ex. `https://example.instructure.com`
-API_KEY = ''  # Canvas API Key
+# Canvas Instance URL ex. `https://example.instructure.com`
+CANVAS_URL = "https://example.com"
+API_KEY = "CHANGEME"  # Canvas API Key
```

Comment:
Any reason this comment isn't on a new line like the others?

Reply:
Mostly I just moved the comments that made the line too long. I can move this one too